libs/providers/flagd/project.json

```diff
@@ -78,8 +78,8 @@
       "executor": "nx:run-commands",
       "options": {
         "commands": [
-          "npx jest --setupFiles './setup-rpc-provider.ts'",
-          "npx jest --setupFiles './setup-in-process-provider.ts'"
+          "npx jest --setupFiles './setup-rpc-provider.ts' --runInBand --detectOpenHandles",
+          "npx jest --setupFiles './setup-in-process-provider.ts' --runInBand --detectOpenHandles"
```

Comment:
These additional flags cause jest to hang (which is what we want) instead of forcing an exit, so we'll detect future regressions with grpc resource hangs.